css/style-min.css

```diff
@@ -9,13 +9,13 @@
   /* width: 320px; */
 }
 
-.container {
-  /* width: 320px; */
+/* .container {
+  width: 320px;
 }
 
 .head-container {
-  /* width: 320px; */
-}
+  width: 320px;
+} */
```

Comment:
- [ ] Please, kindly remove this old piece of commented code to improve the cleanliness of the repo 💪

Reply:
Thanks for catching this. I've taken care of the commented out code.